.dev/systemimage/climate_machine_image.jl

```diff
@@ -11,28 +11,26 @@ sysimage_path =
 
 using Pkg
 Pkg.add("PackageCompiler")
+
 using PackageCompiler
 Pkg.activate(joinpath(@__DIR__, "..", ".."))
 Pkg.instantiate()
 pkgs = Pkg.installed()
```

Comment:
```suggestion
pkgs = VERSION < v"1.4" ? keys(Pkg.installed()) : [v.name for v in values(Pkg.dependencies())]
```
then change the use of `keys(pkgs)` below to just `pkgs`.